brigade/static/scss/molecules/_card.scss

```diff
@@ -1,11 +1,24 @@
 @import '../core/variables';
 @import '../core/mixins';
 
+$card-gutter: 1rem;
 //
 // Card
 //
 
 // TODO: upstream this into the v4 style guide
+// TODO: Remove <a> selectors from all these cards
+a.card--horizontal {
+  align-items: center;
+  display: flex;
+
+  .card-image-container {
```

Comment:
I'm guessing this is inherited from the v4 style guide, but eventually we can have a more-BEM class name like `.card__image`.